src/components/Greeting.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getRandomGreeting } from '../redux/greetings';
+
+function Greeting() {
+  const greeting = useSelector((state) => state.greeting);
+  const error = useSelector((state) => state.error);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getRandomGreeting());
+  }, [dispatch]);
+
+  return (
+    <section className="page">
+      <h1>Random Greeting</h1>
+      <h2>{greeting}</h2>
+      <Link to="/">
+        <button type="button">Back to Home</button>
```

Comment:
After enabling CORS, a random message is displayed. Well done!!! 💯 

![image](https://github.com/IndieCoderMM/hello-react-front-end/assets/38088818/59975b5c-c932-4c01-b66e-794e9283e762)

[OPTIONAL] The failed to fetch error is always displayed even when the random message was fetched successfully. Kindly look into that.

![image](https://github.com/IndieCoderMM/hello-react-front-end/assets/38088818/9dba24f6-3ef3-4246-bc05-f6e39d5fce99)

Reply:
Thanks for pointing this out! I've fixed this error message.